plugins/view-secret.yaml

```diff
@@ -3,19 +3,43 @@ kind: Plugin
 metadata:
   name: view-secret
 spec:
+  version: "v0.1.0"
+  homepage: https://github.com/elsesiy/kubectl-view-secret
+  shortDescription: |
+    Kubernetes CLI plugin to decode Kubernetes secrets
+  description: |+2
+    Let's you conveniently base64 decode all key/value pairs in a given secret.
+
+    $ kubectl view-secret <secret>
```

Comment:
Out of curiosity: do you support decoding certain data.* keys through a cli arg?

Reply:
Currently not, I thought of adding this and just today someone actually asked for it so I probably ship it in the next version 👍